tealium/__init__.py

```diff
@@ -99,40 +105,85 @@ def enable(self):
             statusResult = False
         return statusResult
 
-    def trackEvent(self, title, data=None):
-
+    def trackType(self, eventType, title, data=None, tealiumCallback=None):
```

Comment:
Recommend making this an overloaded function so that 1.0.0 users don't need to change any code to conform to the update.